composer.json

```diff
@@ -25,7 +25,7 @@
         "doctrine/dbal": "^2.9.3",
         "doctrine/event-manager": "^1.1",
         "doctrine/instantiator": "^1.3",
-        "doctrine/persistence": "^1.2",
+        "doctrine/persistence": "^1.3@dev",
```

Comment:
This will be changed in the coming days once we release 1.3, but we need to dogfood here to make sure the deprecation layer works as expected.

Reply:
1.3.0 has been released, this can be changed back to stable.

Reply:
I've put 1.3.1, please restart the build after releasing it.

Reply:
1.3.1 has been released.